Formula/netcdf-mpi.rb

```diff
@@ -0,0 +1,50 @@
+class NetcdfMpi < Formula
+  desc "The NetCDF C I/O library compiled with MPI parallel features enabled"
+  homepage "https://www.unidata.ucar.edu/software/netcdf/"
+  url "https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-c-4.7.4.tar.gz"
+  sha256 "0e476f00aeed95af8771ff2727b7a15b2de353fb7bb3074a0d340b55c2bd4ea8"
+  license "BSD-3-Clause"
+
+  depends_on "hdf5-mpi"
+
+  conflicts_with "netcdf", :because => "netcdf-mpi is an MPI parallel build, use either netcdf or netcdf-mpi"
+
+  def install
+    # use the C compiler wrapper
+    ENV["CC"] = "mpicc"
+
+    # move files that NetCDF wants to place in libdir,
+    # because brew does not want them there. etc was
+    # recommended during the code review
+    inreplace "./Makefile.in",
+          "settingsdir = $(libdir)",
+          "settingsdir = #{etc}/netcdf-mpi"
```

Comment:
```suggestion
          "settingsdir = #{pkgshare}"
```
Are these files intended to be modified by the user? If not, they ought to go in `pkgshare`.

Reply:
agree. not sure why @SMillerDev wanted me to change it to #{etc}.

Reply:
Are these files intended to be modified by the user? Sean probably assumed they were. I suspect they're not.